Assignments/Sorting/SelectionSort.playground/Pages/Selection Sort.xcplaygroundpage/Contents.swift

```diff
@@ -11,17 +11,35 @@ Better define an inner function to find the next minimum in the array.
 
 func selectionSort(array: [Int]) -> [Int] {
     // You may declare array argument with var keyword so that it is copied.
-
-    return array
+    var anArray = array
+    var index = 0
+    while (!anArray.isSorted()) {
```

Comment:
Hi @dingua. Thank you for the PR. Note that `isSorted` is just given to test the result rather than to be used in the solution itself. You may want to check the other PRs.